legacy/src/main/resources/org/pentaho/amazon/s3/messages/messages_en_US.properties

```diff
@@ -1,47 +1,47 @@
-S3FileOutput.Category=Output
-
-S3FileOutput.Injection.AWS_ACCESS_KEY=S3 Access key
-S3FileOutput.Injection.AWS_SECRET_KEY=S3 Secret key
-S3FileOutput.Injection.FILENAME=The filename
-S3FileOutput.Injection.RUN_AS_COMMAND=Run this as command instead? (Y/N)
-S3FileOutput.Injection.PASS_TO_SERVLET=Pass output to servlet? (Y/N)
-S3FileOutput.Injection.CREATE_PARENT_FOLDER=Create parent folder? (Y/N)
-S3FileOutput.Injection.FILENAME_IN_FIELD=Accept filename from field? (Y/N)
-S3FileOutput.Injection.FILENAME_FIELD=The filename field
-S3FileOutput.Injection.EXTENSION=The file extension
-S3FileOutput.Injection.INC_STEPNR_IN_FILENAME=Include stepnr in filename? (Y/N)
-S3FileOutput.Injection.INC_PARTNR_IN_FILENAME=Include partition nr in filename? (Y/N)
-S3FileOutput.Injection.INC_DATE_IN_FILENAME=Include date in filename? (Y/N)
-S3FileOutput.Injection.INC_TIME_IN_FILENAME=Include time in filename? (Y/N)
-S3FileOutput.Injection.SPECIFY_DATE_FORMAT=Specify date time format for filename? (Y/N)
-S3FileOutput.Injection.DATE_FORMAT=Date time format for filename
-S3FileOutput.Injection.ADD_TO_RESULT=Add filenames to result? (Y/N)
-S3FileOutput.Injection.APPEND=Append if file exists? (Y/N)
-S3FileOutput.Injection.SEPARATOR=The separator
-S3FileOutput.Injection.ENCLOSURE=The enclosure
-S3FileOutput.Injection.FORCE_ENCLOSURE=Force the enclosure around fields? (Y/N)
-S3FileOutput.Injection.DISABLE_ENCLOSURE_FIX=Disable the enclosure fix? (Y/N)
-S3FileOutput.Injection.HEADER=Include header row? (Y/N)
-S3FileOutput.Injection.FOOTER=Include footer row? (Y/N)
-S3FileOutput.Injection.FORMAT=The file format line termination? (DOS, UNIX, CR, None)
-S3FileOutput.Injection.COMPRESSION=The compression? (GZip, Hadoop-Snappy, Snappy, Zip, None)
-S3FileOutput.Injection.ENCODING=Encoding type (for allowed values see: http://wiki.pentaho.com/display/EAI/Text+File+Output)
-S3FileOutput.Injection.RIGHT_PAD_FIELDS=Right pad fields? (Y/N)
-S3FileOutput.Injection.FAST_DATA_DUMP=Fast data dump? (Y/N)
-S3FileOutput.Injection.SPLIT_EVERY=Split every ... rows
-S3FileOutput.Injection.ADD_ENDING_LINE=Add ending line after last row
-S3FileOutput.Injection.OUTPUT_FIELDS=The fields to output
-S3FileOutput.Injection.OUTPUT_FIELDNAME=Field to output
-S3FileOutput.Injection.OUTPUT_TYPE=Field type (for allowed values see: http://wiki.pentaho.com/display/EAI/Text+File+Output)
-S3FileOutput.Injection.OUTPUT_FORMAT=Field format
-S3FileOutput.Injection.OUTPUT_LENGTH=Field length
-S3FileOutput.Injection.OUTPUT_PRECISION=Field precision
-S3FileOutput.Injection.OUTPUT_CURRENCY=Field currency symbol
-S3FileOutput.Injection.OUTPUT_DECIMAL=Field decimal symbol
-S3FileOutput.Injection.OUTPUT_GROUP=Field grouping symbol
-S3FileOutput.Injection.OUTPUT_TRIM=Field trim type (none,left,both,right)
-S3FileOutput.Injection.OUTPUT_NULL=Value to replace nulls with
-S3FileOutput.Injection.DO_NOT_CREATE_FILE_AT_STARTUP=Do not create file at start
-
+S3FileOutput.Name=S3 file output
```

Comment:
Why so many changes?
By any chance has an encoding difference occurred?

Reply:
I think something is up with github; I've seen that happen in other PRs recently.  Look a PGBulkLoaderMeta.java here:
https://github.com/pentaho/pentaho-kettle/pull/8347/files

Reply:
I agree with @peterrinehart, and I think this happened because between 8.3 and the cherry pick, the file has been moved elsewhere.
We can see that GitHub, while signaling all lines as changed (removed and added again), it correctly identifies the **actual** changes introduced for "S3FileOutput.Injection.ENCODING" (line 30) and "S3FileOutput.Injection.OUTPUT_TYPE" (line 37).